test/test_transforms.py

```diff
@@ -130,6 +130,20 @@ def test_ten_crop(self):
                 assert len(results) == 10
                 assert expected_output == results
 
+    def test_randomresized_params(self):
+        height = random.randint(24, 32) * 2
+        width = random.randint(24, 32) * 2
+        img = torch.ones(3, height, width)
+        to_pil_image = transforms.ToPILImage()
+        img = to_pil_image(img)
+        size = 100
+        aspect_ratio_range = (0.75, 1.3)
+        scale_range = (0.08, 1)
+        randresizecrop = transforms.RandomResizedCrop(size, scale_range, aspect_ratio_range)
+        _, _, h, w = randresizecrop.get_params(img, scale_range, aspect_ratio_range)
+        aspect_ratio_obtained = w / h
```

Comment:
For python2 compatibility
```suggestion
        aspect_ratio_obtained = float(w) / h
```

Reply:
Great! that should be a bit faster, too.